monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -53,21 +47,35 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
       state.acquisitionData
     )
 
-  def createStripePaymentMethod(stripe: StripePaymentFields, stripeService: StripeService): Future[CreditCardReferenceTransaction] =
+  def createStripePaymentMethod(stripe: StripePaymentFields, stripeService: StripeService): Future[CreditCardReferenceTransaction] = {
+    putCloudWatchCreatePaymentMethodMetric("stripe")
     stripeService
       .createCustomer(stripe.userId, stripe.stripeToken)
       .map { stripeCustomer =>
         val card = stripeCustomer.card
         CreditCardReferenceTransaction(card.id, stripeCustomer.id, card.last4,
           CountryGroup.countryByCode(card.country), card.exp_month, card.exp_year, card.zuoraCardType)
       }
+  }
 
-  def createPayPalPaymentMethod(payPal: PayPalPaymentFields, payPalService: PayPalService): Future[PayPalReferenceTransaction] =
+  def createPayPalPaymentMethod(payPal: PayPalPaymentFields, payPalService: PayPalService): Future[PayPalReferenceTransaction] = {
+    putCloudWatchCreatePaymentMethodMetric("paypal")
     payPalService
       .retrieveEmail(payPal.baid)
       .map(PayPalReferenceTransaction(payPal.baid, _))
+  }
+
+  private def putStartupMetric(paymentType: Either[StripePaymentFields, PayPalPaymentFields]) =
+    if (paymentType.isLeft)
+      putCloudWatchStartupMetric("stripe")
+    else
+      putCloudWatchStartupMetric("paypal")
+
+  def putCloudWatchStartupMetric(paymentMethod: String): Future[Unit] =
+    new RecurringContributionsMetrics(paymentMethod, "monthly")
+      .putContributionSignUpStartProcess().recover({ case _ => () })
 
-  def putCloudWatchMetrics(paymentMethod: String): Future[Unit] =
+  def putCloudWatchCreatePaymentMethodMetric(paymentMethod: String): Future[Unit] =
     new RecurringContributionsMetrics(paymentMethod, "monthly")
       .putContributionSignUpStartProcess().recover({ case _ => () })
```

Comment:
Should this be: putCreatePaymentMethod()?

Reply:
That does sound better. Good idea!

Reply:
I would remove CloudWatch from all the putX method names I think it's redundant